src/test/java/api/support/fixtures/LostItemFeePoliciesFixture.java

```diff
@@ -31,10 +31,6 @@ public LostItemFeePolicyBuilder facultyStandardPolicy() {
     patronBilledAfterAgedLost.put("duration", 12);
     patronBilledAfterAgedLost.put("intervalId", "Months");
 
-    JsonObject chargeAmountItem = new JsonObject();
-    chargeAmountItem.put("chargeType", "anotherCost");
-    chargeAmountItem.put("amount", 5.00);
-
     JsonObject lostItemChargeFeeFine = new JsonObject();
     lostItemChargeFeeFine.put("duration", "6");
     lostItemChargeFeeFine.put("intervalId", "Months");
```

Comment:
Does circulation use this part of the lost item fee policy?

Reply:
Not yet I think.

Reply:
Could it be removed from the fixture for the time being then?

Reply:
Removed